index.js

```diff
@@ -9,27 +9,26 @@ import * as path from 'path';
 function koaDevware (compiler, options) {
   const dev = devMiddleware(compiler, options);
 
-  function middleware (context, next) {
+  function waitUntilValid() {
     return new Promise((resolve, reject) => {
       dev.waitUntilValid(() => {
-        resolve(true);
+        resolve();
       });
 
       compiler.plugin('failed', (error) => {
         reject(error);
       });
-
-      dev(context.req, {
-        end: (content) => {
-          context.body = content;
-        },
-        setHeader: context.set.bind(context)
-      }, next);
     });
   }
 
-  return async (context, next) => {
-    await middleware(context, next);
+  return async (ctx, next) => {
```

Comment:
Sure, sorry I'll change it back. But isn't it good to follow the koa naming conventions? `(ctx, next)`